devicetypes/smartthings/zwave-lock-without-codes.src/zwave-lock-without-codes.groovy

```diff
@@ -405,6 +405,10 @@ private def handleBatteryAlarmReport(cmd) {
 	def deviceName = device.displayName
 	def map = null
 	switch (cmd.zwaveAlarmEvent) {
+		case 0x01: //power has been applied, check if the battery level updated
```

Comment:
nit: Could add `log.debug "Batteries replaced. Queueing a battery get."` as was done for `zwave-lock.groovy`.